src/tokenized-line.coffee

```diff
@@ -47,10 +57,15 @@ class TokenizedLine
       tokenStartColumn += token.screenDelta
 
     if token.isAtomic and tokenStartColumn < column
-      if skipAtomicTokens
+      if clipToAtomicTokenEdge == 'right'
         tokenStartColumn + token.screenDelta
-      else
+      else if clipToAtomicTokenEdge == 'left'
         tokenStartColumn
+      else #'closest'
+        if column > tokenStartColumn + token.screenDelta / 2
```

Comment:
Is this the correct logic? The expression will be evaluated like this:

```coffee
column > tokenStartColumn + (token.screenDelta / 2)
```

Shouldn't we sum first and then divide by 2? If the answer is yes, then our tests should be revisited to shield against this kind of error.